src/core/parser/constants.ts

```diff
@@ -0,0 +1,10 @@
+export const STATIC_OPENING_TAG_REGEX = /<\s*([\w:-]+)([^>]*?)(\/?)\>/
+export const STATIC_CLOSING_TAG_REGEX = /<\/\s*([\w:-]+)\s*>/
+
+export const COMMON_ENTITIES: Record<string, string> = {
+	lt: "<",
+	gt: ">",
+	amp: "&",
+	quot: '"',
+	apos: "'",
+}
```

Comment:
I do not think they should be hard coded, it should probably use a proper existing library for handling html entities in general.